### ISSUE TITLE
YAMLLoadWarning: calling yaml.load

### ISSUE THREAD

**Issue by mbarkhau** [rochacbruno/flasgger#303]
I see that this issue was recently fixed by @scolby33 with ee62207d9671e848ab264900e7809a1dc0876964

The warning reported is the following.

```
lib/python3.7/site-packages/flasgger/utils.py:556: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default
Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  swag = yaml.load(full_doc[yaml_sep + 4:])
```

I'm mainly opening this for others who run into this issue. I'm avoiding the warning by installing from git `-e git+https://github.com/rochacbruno/flasgger.git#egg=flasgger`.

This issue can be closed, or left open until there is a new package published on pypi, not sure what your policy is for this.

**Comment by alessap** [comment]
I have also experienced the same issue. Hoping for a new version with the fix applied to come soon!

**Comment by 40min** [comment]
It will be enough just to downgrade version of PyYAML lib:
```bash
pip uninstall PyYAML
pip install PyYAML==3.13
```

**Comment by alessap** [comment]
> It will be enough just to downgrade version of PyYAML lib:
> 
> ```shell
> pip uninstall PyYAML
> pip install PyYAML==3.13
> ```

Thank you @40min for the suggestion. I can confirm that this does remove the warning! :D

**Comment by metmirr** [comment]
In the `utils.py` file instead of using `yaml.load()` using `yaml.safe_load()` remove warning.

**Comment by mbarkhau** [comment]
@metmir yes that is the fix and it's already in master, it's just not published to pypi yet.
